{"status":"OK","result":{"originalLocale":"en","allowViewHistory":false,"creationTimeSeconds":1417005834,"rating":44,"authorHandle":"I_love_Hoang_Yen","modificationTimeSeconds":1417011254,"id":14877,"title":"\u003cp\u003eisProbablePrime() and nextProbablePrime() not working on Codeforces\u003c/p\u003e","locale":"en","content":"\u003cdiv class\u003d\"ttypography\"\u003e\u003cp\u003eToday when I tried to solve a problem, I found that I\u0027m getting Runtime Error because of using nextProbablePrime() and isProbablePrime().\u003c/p\u003e\u003cp\u003eSee:\u003c/p\u003e \u003cul\u003e   \u003cli\u003eAccepted submission: \u003ca href\u003d\"/contest/396/submission/8859588\" title\u003d\"Submission 8859588 by I_love_Hoang_Yen\"\u003e8859588\u003c/a\u003e\u003c/li\u003e   \u003cli\u003eRuntime Error submission: \u003ca href\u003d\"/contest/396/submission/8859569\" title\u003d\"Submission 8859569 by I_love_Hoang_Yen\"\u003e8859569\u003c/a\u003e\u003c/li\u003e \u003c/ul\u003e\u003cp\u003eThe only difference is in Accepted submission, I used my own nextProbablePrime().\u003c/p\u003e\u003cp\u003ePreviously, I read on Codeforces that \u003ca class\u003d\"rated-user user-admin\" href\u003d\"/profile/MikeMirzayanov\" title\u003d\"Headquarters, MikeMirzayanov\"\u003eMikeMirzayanov\u003c/a\u003e solved this issue: \u003ca href\u003d\"/blog/entry/10037\"\u003e/blog/entry/10037\u003c/a\u003e. Though looks like it is broken again.. :(\u003c/p\u003e\u003c/div\u003e","tags":["biginteger","java","random","isprobableprime"]}}